tests/integration/pilot/common/routing.go

```diff
@@ -452,6 +452,44 @@ spec:
 	return []TrafficTestCase{tc}
 }
 
+// useClientProtocolCases contains tests use_client_protocol from DestinationRule
+func useClientProtocolCases(apps *EchoDeployments) []TrafficTestCase {
+	var cases []TrafficTestCase
+	client := apps.PodA
+	destination := apps.PodC[0]
+	cases = append(cases,
+		TrafficTestCase{
+			name:   "use client protocol with h2",
+			skip:   apps.IsMulticluster(), // Not needed for multi cluster.
```

Comment:
Why not? this actually seems really useful to make sure this handles multinetwork hop

Reply:
I think for this feature the goal is to preserve client protocol if specified in destination rule. Multi network can be a next level test.

Reply:
We get multicluster testing for free though. I don't see why we want to opt out of additional coverage?

Reply:
:-). I agree.  I skipped because it seems it needed additional setup.  Let me take a look